### PR TITLE
 changed done added footer

### PR DESCRIPTION
**Pull request by Prince-yohannan**
# Changes
added footer in login and index.php .
icon for facebook and other can be added

### CODE REVIEW

**Comment by Prince-yohannan** [comment]
footer i had added but  if other changes can be made contact me

File: index.php [hunk 1]
```diff
@@ -52,6 +230,55 @@
...
+  <h3>About<span>Eduonix</span></h3>
+
+  <p class="footer-links">
+    <a href="#">Home</a>
+    |
+    <a href="#">Blog</a>
+    |
+    <a href="#">About</a>
+    |
+    <a href="#">Contact</a>
```
Comment (Prince-yohannan) [inline index.php:246]: add link here

**Comment by siv2r** [comment]
I will look into this. Give me few days time :)

**Comment by Prince-yohannan** [comment]
ok thanks

File: index.php (hunk 1 above)
Comment (siv2r) [reply index.php:246]: Hey, wouldn't it be better if we put this in a `css` file? So, that we can includes this footer in other pages also

File: index.php [hunk 2]
```diff
@@ -52,6 +230,55 @@
...
+  </div>
+  <div>
+    <i class="fa fa-envelope"></i>
+    <p><a href="mailto:support@eduonix.com">support@eduonix.com</a></p>
+  </div>
+</div>
+<div class="footer-right">
+  <p class="footer-company-about">
+    <span>About the company</span>
+    We offer training and skill building courses across Technology, Design, Management, Science and Humanities.</p>
```
Comment (siv2r) [inline index.php:272]: This is not what this project does...lol
Do not forget to update after copy-pasting the code :)

**Review by siv2r — CHANGES_REQUESTED** [review]

**Comment by siv2r** [comment]
**Changes required**
- put the footer `css` in a single file so, that it can be included in other pages also
- upload an image/video showing that you tested this code
- use `phpfmt` formatting

**Comment by Prince-yohannan** [comment]
Ok I will do it

File: index.php (hunk 2 above)
Comment (Prince-yohannan) [reply index.php:272]: what upadate u want

**Comment by Prince-yohannan** [comment]
problem is coming in including the pages i don't know why it is happing

**Comment by siv2r** [comment]
> problem is coming in including the pages i don't know why it is happing

What problem are you facing? Can you be more specific so that I can help?

**Comment by siv2r** [comment]
@Prince-yohannan are you working on this issue?

**Comment by siv2r** [comment]
@Prince-yohannan feel free to submit a better PR when you are done

**Pull request closed without merging**